lib/src/model.g.dart

```diff
@@ -232,14 +229,12 @@ abstract class _$PenaltySerializerMixin {
 DartPlatform _$DartPlatformFromJson(Map<String, dynamic> json) =>
     new DartPlatform(
         (json['components'] as List)?.map((e) => e as String)?.toList(),
-        json['uses'] == null
-            ? null
-            : new Map<String, PlatformUse>.fromIterables(
-                (json['uses'] as Map<String, dynamic>).keys,
-                (json['uses'] as Map).values.map((e) => e == null
-                    ? null
-                    : PlatformUse.values
-                        .singleWhere((x) => x.toString() == 'PlatformUse.$e'))),
+        (json['uses'] as Map<String, dynamic>)?.map((k, e) => new MapEntry(
+            k,
+            e == null
+                ? null
+                : PlatformUse.values
+                    .singleWhere((e) => e.toString() == 'PlatformUse.$e'))),
```

Comment:
The issue **is** in the generated code: Here we have two `e` variables: one from the top-level `map`, one inside the `singleWhere`, and with that, `(e) => e.toString() == 'PlatformUse.$e')` is evaluated as false all the time.

@kevmoo: do we have a quick fix for this, or shall I downgrade the versions?

Reply:
Gah! Downgrade. File an issue on json_serial – will fix ASAP

Reply:
https://github.com/dart-lang/json_serializable/issues/180

Reply:
https://github.com/dart-lang/json_serializable/pull/181

Reply:
Publish @isoos !

Reply:
Thanks, this works great!